pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -86,6 +86,15 @@ type PrometheusSpec struct {
 
 	// +optional
 	Ingress *IngressSpec `json:"ingress,omitempty"`
+
+	Container *corev1.Container `json:"container,omitempty"`
+	// +optional
+	Config *Configuration `json:"config,omitempty"`
+}
+
+// Config  is the the desired state of Prometheus Configuration
+type Configuration struct {
```

Comment:
I think only `Configuration` is not clear. What about `PrometheusConfiguration`.